js/tests/unit/tooltip.spec.js

```diff
@@ -974,6 +974,28 @@ describe('Tooltip', () => {
       })
     })
 
+    it('should hide a tooltip when the escape key is pressed', () => {
```

Comment:
For consistency (maybe it'll fix the CI issue as well) I'd recommend writing this test as it's done for the modals when the escape key is pressed: https://github.com/twbs/bootstrap/blob/3150c6985dcf3e5558788cfc89349293ce5ce992/js/tests/unit/modal.spec.js#L384-L407)

It will look like something like that:

```js
it('should hide a tooltip when the escape key is pressed', () => {
  return new Promise(resolve => {
    fixtureEl.innerHTML = '<a href="#" rel="tooltip" title="Another tooltip"></a>'

    const tooltipEl = fixtureEl.querySelector('a')
    const tooltip = new Tooltip(tooltipEl)

    const spy = spyOn(tooltip, 'hide').and.callThrough()

    tooltipEl.addEventListener('shown.bs.tooltip', () => {
      const keyDownEsc = createEvent('keydown')
      keyDownEsc.key = 'Escape'
      document.dispatchEvent(keyDownEsc)
    })

    tooltipEl.addEventListener('hidden.bs.tooltip', () => {
      expect(spy).toHaveBeenCalled()
      resolve()
    })

    tooltip.show()
  })
})
```